<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 10;</string>
  </property>
  <widget class="QPushButton" name="saveconfig">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>71</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
background-color: rgb(197, 197, 197);

border-radius: 10;
</string>
   </property>
   <property name="text">
    <string>保存配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeldrag">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;微软雅黑&quot;;
border: 3px dotted green;
border-radius: 10;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;drag&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelupload">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>551</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border: 3px dotted green;
border-radius: 10;
font: 75 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:11pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;upload&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>131</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: yellow;
selection-background-color: blue;
border-radius: 10;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>484</x>
     <y>420</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 9pt &quot;微软雅黑&quot;;
background-color: rgb(197, 197, 197);

border-radius: 10;
</string>
   </property>
   <property name="text">
    <string>生成外链并且复制到黏贴板</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>51</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="src">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 255, 5);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;同步目录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bucket">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 255, 5);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;七牛空间&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="AK">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 255, 5);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;AK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="SK">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(138, 255, 5);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;SK&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPlainTextEdit" name="bucketEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="akEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="skEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="srcEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font: 75 9pt &quot;微软雅黑&quot;;
border-radius: 5;
</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
